fades/helpers.py

```diff
@@ -48,3 +50,46 @@ def get_basedir():
         logger.debug("Package xdg not installed; using ~/.fades folder")
         from os.path import expanduser
         return expanduser("~/.fades")
+
+
+def _get_interpreter_info(interpreter=None):
+    """Returns the interpreter's full path using pythonX.Y format and his version"""
+    if interpreter is None:
+        # If interpreter is None by default returns the current interpreter data.
+        major, minor = sys.version_info[:2]
+        executable = sys.executable
+    else:
+        cmd = ("import sys, json; d = dict(path=sys.executable);"
+               "d.update(zip('major minor micro releaselevel serial'.split(), sys.version_info));"
+               "print(json.dumps(d))")
+        args = [interpreter, '-c', cmd]
+        try:
+            requested_interpreter_info = logged_exec(args)
+        except Exception as error:
+            logger.error("Error getting requested interpreter version: %s", error)
+            exit()
+        requested_interpreter_info = json.loads(requested_interpreter_info[0])
+        executable = requested_interpreter_info['path']
+        major = requested_interpreter_info['major']
+        minor = requested_interpreter_info['minor']
+    if executable[-1].isdigit():
+        executable = executable.split(".")[0][:-1]
+    interpreter = "{}{}.{}".format(executable, major, minor)
+    interpreter_version = "{}.{}".format(major, minor)
+    return interpreter, interpreter_version
+
+
+def get_interpreter_version(requested_interpreter):
+    """ Return a 'sanitized' interpreter, its version and indicates if it is the current one. """
+    logger.debug('Getting interpreter version for: %s', requested_interpreter)
+    current_interpreter, current_version = _get_interpreter_info()
+    logger.debug('Current interpreter version is %s', current_version)
+    requested_interpreter, requested_version = _get_interpreter_info(requested_interpreter)
```

Comment:
if requested_interpreter is None (I expect that to be the huge majority of cases) this second _get_interpreter_info call is unnecessary...